packages/progress-bar/src/ProgressBar.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Heading } from "@kaizen/component-library"
+import React, { ReactNode } from "react"
+import classnames from "classnames"
+import styles from "./ProgressBar.scss"
+
+type Props = {
+  value: number
+  max: number
```

Comment:
Sorry I missed this at first, but having a required `max` prop seems like more trouble for consumers than we need, given I'd imagine most cases will be 100.

I just did a bit of searching and came across this as well, sounds like there _could_ be cases where screen readers don't respect a max value that isn't 100:
https://material-ui.com/components/progress/#non-standard-ranges

Reply:
There are plenty of cases in the product (actually more than when it's used as a loader) where the progress bar is not a percentage (see https://cultureamp.design/components/progress-bar/). If value is always a percentage then it shouldn't be named value, it should be `percentage`, or how would you know it's a percentage?  This is the same API as https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress. Btw, I'd considered using that for the implementation but I don't think it'd actually offer any advantage (I'd guess that screen readers would be more likely to respect aria elements than the progress element), and just the hassle of having to undo the browser default styles in  cross browser compatible way (although it'd be the semantically correct choice)

> I just did a bit of searching and came across this as well, sounds like there could be cases where screen readers don't respect a max value that isn't 100
If that's true we can easily normalise the value in the implementation to turn it into a percentage for the aria props (just seems like a limitation of screen readers that we can easily work around)

Reply:
Actually I just realised I was already doing that (kind of unintentionally, it came from the original implementation, but it just made sense to normalise things anyway) - the aria max value is always 100: https://github.com/cultureamp/kaizen-design-system/pull/1852/files#diff-cd39c500d02e475cd6675f1db913712b4d0b876673722bb37e8518e9f59d4eb4R40